scripts/west_commands/sbom/README.rst

```diff
@@ -0,0 +1,277 @@
+produce report for the application and some directory.
+
+
+* You can get an application SBOM from a build directory with the following option:
+
+  .. code-block:: bash
+
+     -d build_directory
+
+  The ``build_directory`` must be already built with the ``west build`` command using
+  ``Ninja`` as underlying build tool (which is default).
+
+  This option requires GNU ``ar`` tool.
+  If you do not have it on your ``PATH``, you can pass it with the ``--ar`` option, for example:
+
+  .. code-block:: bash
+
+     --ar ~/zephyr-sdk/arm-zephyr-eabi/bin/arm-zephyr-eabi-ar
+
+  The command searches for the files used during the build of ``zephyr/zephyr.elf`` target.
+  It also requires ``zephyr/zephyr.map`` file created by the linker.
+
+  .. warning::
+      All the files that are not dependencies of the ``zephyr/zephyr.elf`` target are not taken as an input.
+      If the ``.elf`` file was modified after the linking, those modifications will not be taken.
+
+  If you build directory contains more than one output target or it has a different name,
+  you can add targets after the ``build_directory``.
+  If the ``.map`` file and the associated ``.elf`` file has different names,
+  the ``.map`` file can be provided after the ``:`` sign following the target,
+  for example:
+
+  .. code-block:: bash
+
+     -d build_directory target1.elf target2.elf:file2.map
+
+  .. warning::
+      The ``-d`` option is experimental.
+
+* You can provide list of input files directly on the command line:
+
+  .. code-block:: bash
+
+     --input-files file1 file2 ...
+
+  Each argument of this option can contain globs as defined by `Python's Path.glob`_ with two additions:
+  exclamation mark ``!`` to exclude files and absolute paths.
+
+  For example, if you want to include all ``.c`` files from current directory
+  and all subdirectories recursively:
+
+  .. code-block:: bash
+
+     --input-files '**/*.c'
+
+  Remember to put correct quotes around globs, to make sure that the glob will
+  not be resolved by your shell, but it will go untouched to the command.
+
+  You can prefix pattern with the exclamation mark ``!`` to exclude some files.
+  Patterns are evaluated from left to right, so ``!`` will exclude files from
+  patterns before it, but not after. For example, if you want to include all
+  ``.c`` files from current directory and all subdirectories recursively, except
+  all ``main.c`` files:
+
+  .. code-block:: bash
+
+     --input-files '**/*.c' '!**/main.c'
+
+* You can read list of input files from a file:
+
+  .. code-block:: bash
+
+     --input-list-file list_file
+
+  It does the same as ``--input-files``, but it reads files and patterns from
+  a file (one file or pattern per line). Files and patterns contained in the
+  list file are relative to the list file location (not the current directory).
+  Comments starting with the ``#`` are allowed.
+
+
+.. _west_sbom Specifying output:
+
+Specifying output
+=================
+
```

Comment:
Add some sort of an introductory text here before the bullets. For example something like:
You can specify the format of the report output using the ``output`` argument.